src/main/java/uk/gov/companieshouse/web/accounts/controller/BaseController.java

```diff
@@ -19,4 +27,44 @@ protected void addBackPageAttributeToModel(Model model, String... pathVars) {
 
         model.addAttribute("backButton", Navigator.getPreviousControllerPath(this.getClass(), pathVars));
     }
+
+    /**
+     * Binds one or more API validation errors to model object fields.
+     *
+     * @param bindingResult the binding result object
+     * @param errors        the list of validation errors generated from an
+     *                      API request
+     */
+    protected void bindValidationErrors(BindingResult bindingResult, List<ValidationError> errors) {
```

Comment:
Could we format this file as some of the lines are a bit too long? and I have noticed that happens in other files.